docs/examples/select/use-valueKey.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <el-select-v2
+    v-model="value"
+    :options="options"
+    placeholder="Please select"
+    size="large"
+    value-key="value.name"
+  />
```

Comment:
Maybe you should use `el-select` here, and you can  add the same demo in the select-v2 file.

Reply:
sorry,my bad. this docs should be add in select-v2 

btw, there is a problem like issue #7141 of select, when value is a object input can't display right.
[demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZGl2PlxuICAgIHt7IHZhbHVlIH19XG4gIDwvZGl2PlxuICA8ZWwtc2VsZWN0XG4gICAgdi1tb2RlbD1cInZhbHVlXCJcbiAgICBjbGFzcz1cIm0tMlwiXG4gICAgdmFsdWUta2V5PVwidmFsdWUubmFtZVwiXG4gICAgcGxhY2Vob2xkZXI9XCJTZWxlY3RcIlxuICAgIHNpemU9XCJsYXJnZVwiXG4gID5cbiAgICA8ZWwtb3B0aW9uXG4gICAgICB2LWZvcj1cIml0ZW0gaW4gb3B0aW9uc1wiXG4gICAgICA6a2V5PVwiaXRlbS52YWx1ZVwiXG4gICAgICA6bGFiZWw9XCJpdGVtLmxhYmVsXCJcbiAgICAgIDp2YWx1ZT1cIml0ZW0udmFsdWVcIlxuICAgIC8+XG4gIDwvZWwtc2VsZWN0PlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBsYW5nPVwidHNcIiBzZXR1cD5cbmltcG9ydCB7IHJlZiB9IGZyb20gJ3Z1ZSdcblxuY29uc3QgdmFsdWUgPSByZWYoJycpXG5cblxuY29uc3Qgb3B0aW9ucyA9IFtcbiAge1xuICAgIHZhbHVlOiB7XG4gICAgICBuYW1lOiAnT3B0aW9uMScsXG4gICAgICBpZDogJzEnLFxuICAgIH0sXG4gICAgbGFiZWw6ICdPcHRpb24xJyxcbiAgfSxcbiAge1xuICAgIHZhbHVlOiB7XG4gICAgICBuYW1lOiAnT3B0aW9uMicsXG4gICAgICBpZDogJzInLFxuICAgIH0sXG4gICAgbGFiZWw6ICdPcHRpb24yJyxcbiAgfSxcbiAge1xuICAgIHZhbHVlOiB7XG4gICAgICBuYW1lOiAnT3B0aW9uMycsXG4gICAgICBpZDogJzMnLFxuICAgIH0sXG4gICAgbGFiZWw6ICdPcHRpb24zJyxcbiAgfSxcbiAge1xuICAgIHZhbHVlOiB7XG4gICAgICBuYW1lOiAnT3B0aW9uNCcsXG4gICAgICBpZDogJzQnLFxuICAgIH0sXG4gICAgbGFiZWw6ICdPcHRpb240JyxcbiAgfSxcbiAge1xuICAgIHZhbHVlOiB7XG4gICAgICBuYW1lOiAnT3B0aW9uNScsXG4gICAgICBpZDogJzUnLFxuICAgIH0sXG4gICAgbGFiZWw6ICdPcHRpb241JyxcbiAgfSxcbl1cbnZhbHVlLnZhbHVlID0gb3B0aW9uc1swXS52YWx1ZVxuPC9zY3JpcHQ+IiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319)